examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
```

Comment:
> Given that we advise people to install anaconda, they shoulds have statsmodel
> installed. We could use statsmodel function for that.

No. I've just had last week someone who was using Python in 3D slicer.
Installing any package was quite a lot of work for that person. I am
still trying to limit as much as possible the dependencies, including in
the examples.

However, in a later work, I think that we could integrate such function
in nilearn.plotting. But later.